src/index.html

```diff
@@ -17,6 +29,86 @@
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <a
+        href="https://www.moyo.ua/ua/promo/promo2024/?utm_source=google&utm_medium=cpc&utm_id=19191373185&utm_campaign=Performance_Max_%D0%91%D0%A2_Shopping_ru&gad_source=1&gclid=CjwKCAiArva5BhBiEiwA-oTnXUBkNHbm_M3c085wxz85L2ovMZpt5xMuWtBmEA1uv6qlsVRqwAVughoCMwUQAvD_BwE"
+        class="logo"
+      >
+        <img
+          src="images/logo.png"
+          alt="Logo"
+        />
```

Comment:
Ensure that the `alt` attribute for the `<img>` tag provides a meaningful description of the image. Currently, it is simply 'Logo', which might not be descriptive enough for users relying on screen readers.